common/main.c

```diff
@@ -16,6 +16,9 @@ main(void)
 {
 	console_init(DEV_UART0);
 	dm_init();
+	#if TEST
+	test_crust_boots();
```

Comment:
Please check the acceptance criteria. There should be exactly one function called from `main.c` that runs all tests (so it should be named more generically, like `run_tests()`.

Preprocessor conditions (`#if TEST`) are not indented–they go all the way to the left.

Reply:
done

Reply:
done